### DISCUSSION TITLE
OBS window capture suggestion

### DISCUSSION THREAD

**Discussion by ADSFikosdahfjkijsdah** [obsproject/obs-studio#4457] [Development Questions] [answered]
Hey, 

I wanted to record gameplay of nintendo DS games (On emulator), since the DS screen has 2 separate screens in the same app it's so convenient to have 2 sepatate window captures on the same app. OBS doesn't allow 2 window captures on the same application. I don't understand why you can't just select the same window twice. It's extremely unfortunate. Wouldn't it be possilbe just to allow multiple window captures on 1 app?  Thank you very much

**Comment by Fenrirthviti** [comment, marked as answer]
Please use the forums or Discord for support.
